stylesheets/_layout.sidebar.scss

```diff
@@ -2,7 +2,7 @@
 	display: none;
 	float: none;
 	height: 100%;
-	padding: 0 0 0 40px;
+	// padding: 0 0 0 40px;
```

Comment:
I'm going to be _that_ guy again! If we don't need this line shall we remove it? If it ever needs switching back we can use git to grab the old value. There's a few more instances down below 👇